src/context/auth-context.jsx

```diff
@@ -1,11 +1,17 @@
 import { createContext, useContext, useEffect, useState } from "react";
 import { createUser, getUser } from "../service/user-services";
 import * as auth from "../service/auth-services";
+import React from "react";
```

Comment:
It seems that you are not using the imported `React` variable in the document, but if you need it you could do it in the same line where you import the methods independently.

```suggestion
import React, { createContext, useContext, useEffect, useState } from "react";
import { createUser, getUser } from "../service/user-services";
import * as auth from "../service/auth-services";
```